### PR TITLE
Functional test for customized XML marshal/unmarshal for user defined…

### PR DESCRIPTION
**Pull request by yuhaohaoyu**
… aliases

Signed-off-by: Hao Yu <yuh@us.ibm.com>

**What this PR does / why we need it**:

This PR adds function test to demonstrate and test the effect of the recently merged PR, #4927. The said PR fixed a VMI migration issue where some device aliases in the domain XML were not kept consistent during the migration. 

Following a sound review suggestion, https://github.com/kubevirt/kubevirt/pull/4927#discussion_r573059909, we added a end-to-end migration test case with a `tablet` device in the migratable VMI. The newly added test will carry out a successfully migration with the fix defined in #4927 , but will fail the migration without the fix. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

The added function test directly follows the suggestion at https://github.com/kubevirt/kubevirt/pull/4927#discussion_r573059909, in PR #4927 . 

**Release note**:

```release-note
"NONE"
```

### CODE REVIEW

File: tests/migration_test.go [hunk 1]
```diff
@@ -406,6 +406,38 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 			})
 		})
 		Context("with a Cirros disk", func() {
+			FIt("should be successfully migrate with a tablet device", func() {
```
Comment (xpivarc) [inline tests/migration_test.go:409]: We need to remove the `FIt`.

Comment (yuhaohaoyu) [reply tests/migration_test.go:409]: Sorry for this. Thanks.

**Review by xpivarc — APPROVED** [review]

**Pull request merged**